CalibCalorimetry/HcalAlgos/src/HcalDbHardcode.cc

```diff
@@ -290,7 +290,7 @@ HcalRecoParam HcalDbHardcode::makeRecoParam (HcalGenericDetId fId) {
     //  param1.
     containmentCorrectionFlag = 1;         // p0
     containmentCorrectionPreSample = 0;    // p1
-    float phase  = 25.0;
+    float phase  = 6.0;
     float Xphase = (phase + 32.0) * 4.0;   // never change this line 
                                            //(offset 50nsec, 0.25ns step)
     Iphase       = Xphase;                 // p2
```

Comment:
hello @mariadalfonso

what is the change from 25 to 6 ascribed to? 

At any rate, the chance 25 -> 6 propagates through:

 phase =>  Xphase => Iphase 

which are all local variables to makeRecoParam, 
it has no consequences.